notebooks/playground/rename_wandb_group.ipynb

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
could you you please remove the notebooks and keep them local for now?

Reply:
I would like to keep in git, then we dont have to copy it from trams again.